src/utility.js

```diff
@@ -79,8 +80,8 @@ function parsePaths(source, stylesheets, options) {
         /* Fix the case when there is a query string or hash */
         sheet = sheet.split('?')[0].split('#')[0];
 
-        /* Path already parsed by PhantomJS */
-        if (sheet.substr(0, 5) === 'file:') {
+        /* Path already parsed by PhantomJS, or user passed in local path */
```

Comment:
Would it make sense to make this a big if/else? If http-or-https-url, else if file-url, else X (if there's an X..which maybe there isn't really anymore?).